recipe/patches/0001-GH-41333-C-CMake-Prefer-protobuf-config.cmake-to-Fin.patch

```diff
@@ -16,11 +46,6 @@ index 15fe1b4..503cffe 100644
 -  set(ProtobufAlt_VERSION_MINOR ${Protobuf_VERSION_MINOR})
 -  set(ProtobufAlt_VERSION_PATCH ${Protobuf_VERSION_PATCH})
 -  set(ProtobufAlt_VERSION_TWEEK ${Protobuf_VERSION_TWEEK})
-+  if(Protobuf_PROTOC_EXECUTABLE)
-+    set_target_properties(protobuf::protoc PROPERTIES
-+    IMPORTED_LOCATION_RELEASE "${Protobuf_PROTOC_EXECUTABLE}"
-+  )
-+  endif()
```

Comment:
@raulcd, the patch you checked in didn't match what got committed in https://github.com/apache/arrow/commit/64be7a2d073759b5eea90e3a6167ca2c1d56ce79. I opened a PR for the delta: https://github.com/apache/arrow/pull/41582

Reply:
Yes, the fix for MINGW is what got committed but for crosscompiling on conda I required to tweak that too. Thanks for submitting the patch.